Dockerfile

```diff
@@ -1,38 +1,52 @@
-ARG VERSION=unspecified
+ARG GIT_COMMIT=unspecified
+ARG GIT_REMOTE=unspecified
+ARG VERSION=0.0.1
 
-FROM python:3.9.6-alpine
+FROM python:3.9.6
 
+ARG GIT_COMMIT
+ARG GIT_REMOTE
 ARG VERSION
 
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 # Note: Additional labels are added by the build workflow.
+LABEL git_commit=$GIT_COMMIT
+LABEL git_remote=$GIT_REMOTE
 LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
+LABEL version=$VERSION
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
+ENV PCA_REPORT_TOOLS_SRC="/usr/src/pca-report-tools"
 ENV ECHO_MESSAGE="Hello World from Dockerfile"
```

Comment:
We can just leave it alone for now since it's really an issue with the skeleton.  When we get around to changing it in the skeleton, it will filter down to this repo also.

Reply:
I thought about this a bit more and there's no sense leaving it in here if we're not using it, so just go ahead and delete that `ECHO_MESSAGE` env variable and all references to it.

Reply:
Resolved in b4f38daf8fd9b8a49771e281ca902cee00fdbe0d

Reply:
I believe you can also delete these two lines containing unused variables:
https://github.com/cisagov/pca-report-generator-docker/blob/b4f38daf8fd9b8a49771e281ca902cee00fdbe0d/tests/container_test.py#L11-L12

Reply:
Resolved in c56cbe2147f5fb97242a87075b56fbd7c15129da